src/PasswordResetController.js

```diff
@@ -34,11 +34,12 @@ function (Okta, FormController, FormType, ValidationUtil, FactorUtil, FooterSign
         return ValidationUtil.validatePasswordMatch(this);
       },
       save: function () {
-        var self = this;
```

Comment:
any specific reason for renaming this variable? Just curious since we are not declaring any other ```self``` in this context, and it seems to me ```self``` is the most common practice in this repo..